### PR TITLE
Updated documentation regarding Awspec::DuplicatedResourceTypeError

### PR DESCRIPTION
**Pull request by glasswalk3r**

### CODE REVIEW

**Comment by glasswalk3r** [comment]
Not sure what went wrong with TravisCI:

```
[myself@localhost awspec]$ git branch
  bugfix/typo_in_dynamodb_stub
  bugfix/typo_in_lambda_finder
* doc/advanced_ec2
  feature/sns_resource_type
  feature/testing
  master
[myself@localhost awspec]$ bundle exec rake spec:generator_doc
/home/myself/.rbenv/versions/2.4.4/bin/ruby -I/home/myself/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/rspec-core-3.8.0/lib:/home/myself/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/rspec-support-3.8.0/lib /home/myself/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/rspec-core-3.8.0/exe/rspec --pattern spec/generator/doc/\*_spec.rb
...................................................................

Finished in 1.12 seconds (files took 1.81 seconds to load)
67 examples, 0 failures
```

**Comment by amukiza** [comment]
@glasswalk3r my pull request has the same issue. 

I have tried several things including using a vagrant to run the tests on the same linux version, rvm and ruby version as Travis e.t.c.

File: doc/_resource_types/ec2.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ describe ec2('my-ec2') do
 end
 ```
 
-### advanced
+### Advanced
```
Comment (k1LoW) [inline doc/_resource_types/ec2.md]: Could you use `advanced` for auto document generation ?

**Comment by k1LoW** [comment]
Hi @glasswalk3r  Thank you for document update.

I wrote troubleshooting when CI failed ( https://github.com/k1LoW/awspec/blob/master/doc/contributing.md#troubleshooting )

Could you update aws-sdk-ruby ?

**Comment by glasswalk3r** [comment]
Did a fresh `git clone` and tried the procedure:

```
alceu@yggdrasil:~/Projetos/awspec$ bundle install --path vendor/bundle
Fetching gem metadata from https://rubygems.org/...............
Fetching gem metadata from https://rubygems.org/..
Resolving dependencies....
...
Bundle complete! 6 Gemfile dependencies, 191 gems now installed.
Bundled gems are installed into `./vendor/bundle`
alceu@yggdrasil:~/Projetos/awspec$ bundle update
Fetching gem metadata from https://rubygems.org/...............
Fetching gem metadata from https://rubygems.org/..
Resolving dependencies....
Using rake 12.3.1
Using public_suffix 3.0.3
Using addressable 2.5.2
Using ast 2.4.0
Using aws-eventstream 1.0.1
Using aws-partitions 1.111.0
Using aws-sigv4 1.0.3
Using jmespath 1.4.0
Using aws-sdk-core 3.37.0

alceu@yggdrasil:~/Projetos/awspec$ bundle exec bin/toolbox docgen > doc/resource_types.md
alceu@yggdrasil:~/Projetos/awspec$ git status
No ramo doc/advanced_ec2
Your branch is up to date with 'origin/doc/advanced_ec2'.

Changes not staged for commit:
  (utilize "git add <arquivo>..." para atualizar o que será submetido)
  (utilize "git checkout -- <arquivo>..." para descartar mudanças no diretório de trabalho)

	modified:   doc/resource_types.md

Arquivos não monitorados:
  (utilize "git add <arquivo>..." para incluir o que será submetido)

	vendor/

nenhuma modificação adicionada à submissão (utilize "git add" e/ou "git commit -a")
alceu@yggdrasil:~/Projetos/awspec$ git diff doc/resource_types.md
diff --git a/doc/resource_types.md b/doc/resource_types.md
index 8a95f0d..312bc78 100644
--- a/doc/resource_types.md
+++ b/doc/resource_types.md
@@ -2319,7 +2319,7 @@ end
 ```
 
 
-### its(:vpc_id), its(:db_instance_identifier), its(:db_instance_class), its(:engine), its(:db_instance_status), its(:master_username), its(:db_name), its(:endpoint), its(:allocated_storage), its(:instance_create_time), its(:preferred_backup_window), its(:backup_retention_period), its(:db_security_groups), its(:availability_zone), its(:preferred_maintenance_window), its(:pending_modified_values), its(:latest_restorable_time), its(:multi_az), its(:engine_version), its(:auto_minor_version_upgrade), its(:read_replica_source_db_instance_identifier), its(:read_replica_db_instance_identifiers), its(:read_replica_db_cluster_identifiers), its(:license_model), its(:iops), its(:character_set_name), its(:secondary_availability_zone), its(:publicly_accessible), its(:status_infos), its(:storage_type), its(:tde_credential_arn), its(:db_instance_port), its(:db_cluster_identifier), its(:storage_encrypted), its(:kms_key_id), its(:dbi_resource_id), its(:ca_certificate_identifier), its(:domain_memberships), its(:copy_tags_to_snapshot), its(:monitoring_interval), its(:enhanced_monitoring_resource_arn), its(:monitoring_role_arn), its(:promotion_tier), its(:db_instance_arn), its(:timezone), its(:iam_database_authentication_enabled), its(:performance_insights_enabled), its(:performance_insights_kms_key_id), its(:performance_insights_retention_period), its(:enabled_cloudwatch_logs_exports), its(:processor_features), its(:deletion_protection)
+### its(:vpc_id), its(:db_instance_identifier), its(:db_instance_class), its(:engine), its(:db_instance_status), its(:master_username), its(:db_name), its(:endpoint), its(:allocated_storage), its(:instance_create_time), its(:preferred_backup_window), its(:backup_retention_period), its(:db_security_groups), its(:availability_zone), its(:preferred_maintenance_window), its(:pending_modified_values), its(:latest_restorable_time), its(:multi_az), its(:engine_version), its(:auto_minor_version_upgrade), its(:read_replica_source_db_instance_identifier), its(:read_replica_db_instance_identifiers), its(:read_replica_db_cluster_identifiers), its(:license_model), its(:iops), its(:character_set_name), its(:secondary_availability_zone), its(:publicly_accessible), its(:status_infos), its(:storage_type), its(:tde_credential_arn), its(:db_instance_port), its(:db_cluster_identifier), its(:storage_encrypted), its(:kms_key_id), its(:dbi_resource_id), its(:ca_certificate_identifier), its(:domain_memberships), its(:copy_tags_to_snapshot), its(:monitoring_interval), its(:enhanced_monitoring_resource_arn), its(:monitoring_role_arn), its(:promotion_tier), its(:db_instance_arn), its(:timezone), its(:iam_database_authentication_enabled), its(:performance_insights_enabled), its(:performance_insights_kms_key_id), its(:performance_insights_retention_period), its(:enabled_cloudwatch_logs_exports), its(:processor_features), its(:deletion_protection), its(:listener_endpoint)
```

So I'm guessing that I should use `bundle update && bundle exec bin/toolbox docgen > doc/resource_types.md && git commit -a && git push` as a single step to be sure.

File: doc/_resource_types/ec2.md (hunk 1 above)
Comment (glasswalk3r) [reply doc/_resource_types/ec2.md]: Done!

**Comment by k1LoW** [comment]
@glasswalk3r GOOOOD!!!! Thank you !!!

**Comment by k1LoW** [comment]
Released as v1.12.0.

**Pull request merged**